spring/src/main/java/org/axonframework/spring/config/SpringAxonAutoConfigurer.java

```diff
@@ -134,7 +137,9 @@ private void registerSagaBeanDefinitions(Configurer configurer, BeanDefinitionRe
         for (String saga : sagas) {
             Saga sagaAnnotation = beanFactory.findAnnotationOnBean(saga, Saga.class);
             SagaConfiguration<?> sagaConfiguration = SagaConfiguration.subscribingSagaManager(beanFactory.getType(saga));
-            // TODO: Register resource injector here.
+
+            sagaConfiguration.configureResourceInjector(c -> beanFactory.getBean(ResourceInjector.class));
```

Comment:
Only one little thing, actually. Adding the resourceInjector to the Saga isn't necessary. By default, it takes the injector configured on the global level (which may or may not be the one in the application context). Therefore, line 140 and 141 above can be removed.